Azure/sonic-buildimage
# Update arista driver submodule to includes interrupt handling changes

**- What I did**

- Expose scd interrupt handling addresses in python
- Add interrupts as inventory objects for supported platforms
- Define get_transceiver_change_event function in sfputil

**- How I did it**

**- How to verify it**

- Load image with changes onto device, run get_transceiver_change_event and plug in/out transceivers to see output.

**- Description for the changelog**


Update arista driver submodule to includes interrupt handling changes

**- A picture of a cute animal (not mandatory but encouraged)**
